src/Stripe.net/Services/Issuing/Cardholders/CardholderListOptions.cs

```diff
@@ -7,6 +7,9 @@ public class CardholderListOptions : ListOptionsWithCreated
         [JsonProperty("email")]
         public string Email { get; set; }
 
+        [JsonProperty("is_default")]
+        public bool? IsDefault { get; set; }
+
```

Comment:
Remove that one, it's going away (shouldn't be in the spec, will be removed soon) so let's not add this

Reply:
Will do. Thanks for knowing the future.